ui-library-add-call-recording/api/transfer/BlobStorage.cs

```diff
@@ -37,6 +38,17 @@ public async Task<bool> UploadFileAsync(string blobName, Stream inStream)
                 await blobClient.UploadAsync(inStream);
                 return true;
             }
+
+            public string[] GetBlobs(string prefix)
+            {
+                var pages = this.containerClient.GetBlobs(Azure.Storage.Blobs.Models.BlobTraits.None, Azure.Storage.Blobs.Models.BlobStates.None, prefix);
+                var response = new List<string>();
```

Comment:
nit: response -> blobFileNames

Reply:
done. Also renamed the method to `GetBlobNames`.